pkgs/development/python-modules/ovito/default.nix

```diff
@@ -1,34 +1,43 @@
-{ stdenv, fetchgit
+{ stdenv
+, fetchFromGitLab
 , cmake
-, libav, netcdf, qscintilla, zlib, boost, git, fftw, hdf5, libssh
-, pythonPackages
+, ffmpeg
+, netcdf
+, qscintilla
+, zlib
+, boost
+, git
+, fftw
+, hdf5
+, libssh
+, qt5
+, python
 }:
 
-stdenv.mkDerivation {
-  # compilation error in 2.9.0 https://gitlab.com/stuko/ovito/issues/40
-  # This is not the "released" 3.0.0 just a commit
-  version = "3.0.0";
+stdenv.mkDerivation rec {
+  version = "3.3.1";
   pname = "ovito";
 
-  src = fetchgit {
-    url = "https://gitlab.com/stuko/ovito";
-    rev = "a28c28182a879d2a1b511ec56f9845306dd8a4db";
-    sha256 = "1vqzv3gzwf8r0g05a7fj8hdyvnzq2h3wdfck7j6n1av6rvp7hi5r";
+  src = fetchFromGitLab {
+    owner = "stuko";
+    repo = "${pname}";
```

Comment:
but this is minor
```suggestion
    repo = pname;
```

Reply:
Amended in 46a8d9d82ac51569fefbc4001e832bc799b7ef05